lib/sign.js

```diff
@@ -101,15 +105,6 @@ function sign (program, options, config) {
       return result;
     });
 
-  }).then(function(result) {
-    logger.log(result.success ? 'SUCCESS' : 'FAIL');
-    config.systemProcess.exit(result.success ? 0 : 1);
```

Comment:
because all of these moved, the tests that were checking for process exits failed. I think this patch would be easier f you kept all these as-is but pass in a stub `systemProcess` in your script (I know that's sort of lame but it's easy).